app/client/cli/peer/connections.go

```diff
@@ -0,0 +1,80 @@
+package peer
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"google.golang.org/protobuf/types/known/anypb"
+
+	"github.com/pokt-network/pocket/app/client/cli/helpers"
+	"github.com/pokt-network/pocket/p2p/debug"
+	"github.com/pokt-network/pocket/shared/messaging"
+)
+
+var (
+	connectionsCmd = &cobra.Command{
+		Use:   "connections",
+		Short: "Print open peer connections",
+		RunE:  connectionsRunE,
+	}
+)
+
+func init() {
+	PeerCmd.AddCommand(connectionsCmd)
+}
+
+func connectionsRunE(cmd *cobra.Command, _ []string) error {
+	var routerType debug.RouterType
+
+	bus, err := helpers.GetBusFromCmd(cmd)
+	if err != nil {
+		return err
+	}
+
+	switch {
+	case stakedFlag:
+		if unstakedFlag || allFlag {
+			return ErrRouterType
+		}
+		routerType = debug.StakedRouterType
+	case unstakedFlag:
+		if stakedFlag || allFlag {
+			return ErrRouterType
+		}
+		routerType = debug.UnstakedRouterType
+	// even if `allFlag` is false, we still want to print all connections
+	default:
+		if stakedFlag || unstakedFlag {
+			return ErrRouterType
+		}
+		routerType = debug.AllRouterTypes
+	}
+
+	debugMsg := &messaging.DebugMessage{
+		Action: messaging.DebugMessageAction_DEBUG_P2P_PEER_CONNECTIONS,
+		Type:   messaging.DebugMessageRoutingType_DEBUG_MESSAGE_TYPE_BROADCAST,
+		Message: &anypb.Any{
+			Value: []byte(routerType),
+		},
+	}
+	debugMsgAny, err := anypb.New(debugMsg)
+	if err != nil {
+		return fmt.Errorf("creating anypb from debug message: %w", err)
+	}
+
+	if localFlag {
```

Comment:
So if this is false we don't print anything?

I don't fully understand its use. Is it like a "verbose" flag?

Reply:
This `localFlag` determines whether we print locally ie to the terminal that ran the command or on the node itself AFAIK. With `-l` we will see the output in our terminal, otherwise it will be logged by the node